Source/Membership/MembershipClient.swift

```diff
@@ -21,7 +21,7 @@
 import Foundation
 
 /// Membership HTTP client.
-public class MembershipClient: CompletionHandlerType<Membership> {
+open class MembershipClient: CompletionHandlerType<Membership> {
```

Comment:
Why do we need to change public to open for client API? The third party need to override or inherit them?